Documentation/CodingStyle.md

```diff
@@ -496,24 +496,31 @@ UniqueObject& my_unique_object(); // Free function.
 
 ### Comments
 
-Make comments look like sentences by starting with a capital letter and ending with a period (punctuation). One exception may be end of line comments like this `if (x == y) // false for NaN`.
+Comments should be written using `//` and not `/* */`, except for the copyright notice.
+
+Make comments look like sentences by starting with a capital letter and ending with a period (punctuation). One exception may be end of line comments like this: `if (x == y) // false for NaN`
+
+While we have no fixed rule about line length, please wrap long comments onto multiple lines so that they are easier to read.
```

Comment:
I got a review comment a while back saying that 120 characters was what we try to wrap comments at. So that's what I've been doing. It seems possibly useful to have some documented guidance for a _suggested_ line length to try to keep comments under, if not a hard requirement. But I don't personally feel strongly about it

Reply:
Huh, really? I've literally never been given a number, I just go with what feels right.

Reply:
It's been a loose style pointed out in code reviews, but would be nice to have written down
https://github.com/SerenityOS/serenity/pull/16072#discussion_r1022711773
https://github.com/SerenityOS/serenity/pull/17016#discussion_r1070390922